views/forms/TextInputTopLabel/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { OutlinedInput, InputLabel } from '@mui/material/'
+import { useController } from 'react-hook-form'
+
+const TextInputTopLabel = (props) => {
+  const { errors } = props
+  const { field } = useController(props)
+
+  return (
+    <>
+      <InputLabel
+        aria-label={props.label}
+        shrink
+        {...props.inputLabel}
+        htmlFor={props.name}
+      >
+        {props.label}
+      </InputLabel>
+      <OutlinedInput
+        {...field}
+        aria-invalid={!!errors ? 'true' : 'false'}
+        id={props.name}
+        fullWidth={props.fullWidth}
+        error={!!errors}
+        inputProps={{ ...props.inputProps }}
+        margin={props.margin || 'none'}
+        required={props.required}
+        type={props.type}
+        sx={props.sx}
+        size={props.size}
+      />
+      {!!errors?.message && (
+        <InputLabel
```

Comment:
do we need both input labels or can we conditionally send different props to the existing one?

Reply:
The top input label is the one that sits above the input element, the input label on the bottom is the error display, I went with the input label because it has an error state, and the it defaults to the error color etc...

<img width="519" alt="Screenshot 2023-11-15 at 2 59 21 PM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/562671c9-ad07-43c4-bf4c-b311d08446ce">

Reply:
I think we can use the input's `helperText` prop for this.

Reply:
I was hoping that it would have it like the TextField, I learned that the [OutlinedInput](https://mui.com/material-ui/api/outlined-input/) is missing that prop

Reply:
The [MUI `TextField` component](https://mui.com/material-ui/react-text-field/#components) uses the `OutlinedInput` as well as other components, so I think we should use that instead and leverage the `helperText` prop. We might also be able to remove the `InputLabel`.

![Screenshot 2023-11-16 at 9 40 05 AM](https://github.com/AMP-SCZ/dpdash/assets/2905145/e0d2d4f1-5b45-4736-8ee6-44cdf1f37e2d)

Reply:
When I used the variant of outline on the TextField component the top left side where the label sits is notched. We can add the notch to the OutlinedInput but just to close that notch on the TextField component is getting weird.

Reply:
ok, in that case we should use the [`FormHelperText`](https://mui.com/material-ui/api/form-helper-text/) component for the helper text 👍

Reply:
[2b10634](https://github.com/AMP-SCZ/dpdash/pull/478/commits/2b10634c506de607903b3c18f8cbfd56e8904186)